HW_7/logger.py

```diff
@@ -0,0 +1,11 @@
+from datetime import datetime
+
+
+def log(func):
+    def wapper(*args):
+        with open('log.csv', 'a', encoding='utf-8') as log_file:
+            log_file.write(f'{datetime.now()} - запущена функция' 
+                            '{func.__name__} {"с аргументами" if args else "без аргументов"}' 
```

Comment:
Тут у меня не было дописано верно. И вы, соответственно, тоже не дописали.
```suggestion
                            '{func.__name__} {f"с аргументами ({args})" if args else "без аргументов"}' 
```